src/table.js

```diff
@@ -8,6 +8,14 @@ var Table = function (protoTable) {
       callback(entry);
     }
   };
+
+  this.asyncEntries = async function (callback) {
```

Comment:
Instead of making a new function, can we just check if the callback is an async function and handle it accordingly?

For example
```
const AsyncFunction = (async () => {}).constructor;

if(callback instanceof AsyncFunction) {
}
```

Reply:
Thank you for the suggestion, I am new to javascript, so that is helpful.  I have made the suggested changes.